### DISCUSSION TITLE
article/notion-next-cloud-flare

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#83] [Announcements]
# article/notion-next-cloud-flare

无需服务器、即使是小白也能几分钟搭建自己的独立博客站～如果你在使用Notion这款神级笔记本的话，不妨来试试顺手建个网站🤣🤣🤣，这是一款基于NotionAPI的博客系统。

https://docs.tangly1024.com/about

**Comment by GD2021** [comment]
用CloudFlarePage部署报错：
TypeError: Cannot read properties of undefined (reading 'map')
    at getStaticPaths (/opt/buildhome/repo/.next/server/pages/[...slug].js:389:25)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async buildStaticPaths (/opt/buildhome/repo/node_modules/next/dist/build/utils.js:699:33)
    at async /opt/buildhome/repo/node_modules/next/dist/build/utils.js:1127:113
    at async Span.traceAsyncFn (/opt/buildhome/repo/node_modules/next/dist/trace/trace.js:103:20)
请问怎么回事，如何解决呢？

**Comment by luckypoem** [comment]
hi,如何获得NOTION_PAGE_ID的值？？

**Reply by luckypoem** [reply]
哦，我duplicate了这个
https://craigary.notion.site/866916e3b939468b9b6f1d47dce99f9c?v=73fd891cab654e35b44893608a23760e，得到自己的：
https://luckypoem.notion.site/luckypoem/e31b3f462dfb47fab104277ed48050ea?v=7c86f37e0f404790ae8435d1301ebfc8
然后知道我的NOTION_PAGE_ID的值是e31b3f462dfb47fab104277ed48050ea

**Reply by luckypoem** [reply]
部署成功：https://notionnext-b0l.pages.dev/

**Comment by luckypoem** [comment]
不过如何发帖呢？

**Comment by luckypoem** [comment]
你的网站模板地址是什么？像这样的模板地址：https://craigary.notion.site/866916e3b939468b9b6f1d47dce99f9c?v=73fd891cab654e35b44893608a23760e
